bin/varnishtest/tests/v00005.vtc

```diff
@@ -32,6 +32,17 @@ varnish v1 -vcl {
 	}
 }
 
+# Check expect_close definition
+varnish v1 -vcl {
+	backend b1 {
+		.host = "${localhost}";
+		.probe = {
+			.url = "/";
+			.expect_close = false;
+		}
+	}
+}
```

Comment:
I just realized that you are testing the happy path here, and what I had requested was invalid code:

```diff
--- bin/varnishtest/tests/v00005.vtc
+++ bin/varnishtest/tests/v00005.vtc
@@ -33,12 +33,12 @@ varnish v1 -vcl {
 }
 
 # Check expect_close definition
-varnish v1 -vcl {
+varnish v1 -errvcl {Expected "true" or "false"} {
        backend b1 {
                .host = "${localhost}";
                .probe = {
                        .url = "/";
-                       .expect_close = false;
+                       .expect_close = faux;
                }
        }
 }
```